Lombiq.Tests.UI.Shortcuts/Controllers/ShiftTimeController.cs

```diff
@@ -0,0 +1,29 @@
+using Lombiq.Tests.UI.Shortcuts.Services;
+using Microsoft.AspNetCore.Mvc;
+using OrchardCore.Modules;
+using System;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Lombiq.Tests.UI.Shortcuts.Controllers;
+
+[SuppressMessage(
+    "Major Code Smell",
+    "S6967:ModelState.IsValid should be called in controller actions",
+    Justification = "Not relevant in a test-only controller.")]
+public class ShiftTimeController : Controller
```

Comment:
Any particular reason why almost all of the identifiers are using the action verb "shift time" instead of the operation name "time shift"?

Reply:
The reason is that I did it by accident.

Reply:
OK then, let's fix this here before releasing this feature or demoing it publicly: https://github.com/Lombiq/UI-Testing-Toolbox/issues/422